### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-270.1][WIP]: Icon: UI cannot be accessed by Keyboard Tab

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11290]
#### Is this a bug, enhancement, or feature request?
Bug
#### Describe your proposal.
Component having issue:https://sap.github.io/fundamental-ngx/#/core/icon
Each & every UI, must be accessible using the keyboard.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Take any icon. Try to reach it by Keyboard Tab. It won't be reachable.
#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by droshev** [comment]
@JyotiKumari1507 Can you provide more information

**Comment by JyotiKumari1507** [comment]
@droshev  As per accessibility team any icon component which we use should be accessible by keyboard. We can discuss on this more in next call

**Comment by droshev** [comment]
@JyotiKumari1507 we need the input written so it will be easier to process

**Comment by JyotiKumari1507** [comment]
@droshev  refer screenshot shared by accessibility team. As per them in this case 'The 'Communication' icon not reachable using the keyboard. As per them Each & every UI, must be accessible using the keyboard.
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/b9f2e314-2c3d-4daf-b60e-bd0419bb1006">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 please share stackblitz example

**Comment by droshev** [comment]
@JyotiKumari1507 We didn't get updates for some time so we are closing the issue. If you think there is a bug, please, create a stackblitz example, document the steps and what the expectation is in a new ticket.

**Comment by divyamalhotra488** [comment]
Hi @droshev, we were able to resolve this from the application side. Thanks.
